source/switching_to_live/index.html.md.erb

```diff
@@ -51,7 +51,20 @@ Choose which card types to accept by signing into the [GOV.UK Pay admin tool](ht
 
 If you use Worldpay, SmartPay or ePDQ, you must also make sure the same card types are selected in your PSP account.
 
-### If you use Worldpay
+### Block prepaid cards
+
+You can block users from paying with prepaid cards.
+
+You must register with Visa to get permission first. To register, either:
+
+- [contact us](/support_contact_and_more_information/#contact-us) if you use Stripe
+- contact your PSP
+
+After you've registered with Visa, [contact us](/support_contact_and_more_information/#contact-us) to block prepaid cards on your account.
+
+We usually cannot block Mastercard cards, because we cannot always tell if they are prepaid or not.
```

Comment:
Do you think we need to be more explicit about the implications of this? That is, that if you block prepaid cards, most prepaid Mastercard cards will still be accepted.